presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMinMaxByAggregation.java

```diff
@@ -142,14 +141,8 @@ public void testMaxDoubleDouble()
                 new Signature("max_by", AGGREGATE, parseTypeSignature(StandardTypes.DOUBLE), parseTypeSignature(StandardTypes.DOUBLE), parseTypeSignature(StandardTypes.DOUBLE)));
         assertAggregation(
                 function,
-                null,
-                createDoublesBlock(null, null),
-                createDoublesBlock(null, null));
-
-        assertAggregation(
-                function,
                 2.0,
-                createDoublesBlock(3.0, 2.0, null),
+                createDoublesBlock(3.0, 2.0, 1.0),
```

Comment:
This is not what I meant, @rongrong.

You still need to test null values for non-nullable function positions. For aggregation functions, non-nullable means that the value will be skipped.